src/components/layout/TradingLayout/variablesCss.ts

```diff
@@ -0,0 +1,88 @@
+import { css } from 'styled-components'
+
+const AllColors = `
```

Comment:
This is a copy pasta from v1. @biocom do we need all this?
Maybe we can start smaller and add on need?

Also, I kind of like better the approach of uniswap on how they define the colors, and having 2 global styles, one for dark/light and one is just the defaults/css-reset

Reply:
> do we need all this?
We might. I'd suggest to just get the file in place and then decide what we need/how we want to structure colors while working on the rest of the components.

Reply:
> do we need all this?

We might. I'd suggest to just get the file in place and then decide what we need/how we want to structure colors while working on the rest of the components.